tools/repl_test.py

```diff
@@ -137,6 +143,18 @@ def test_lexical_scoped_variable(self):
         self.assertEqual(err, '')
         self.assertEqual(code, 0)
 
+    def test_missing_deno_dir(self):
+        current_deno_dir = os.environ["DENO_DIR"]
+        missing_deno_dir = os.path.join(current_deno_dir, 'missing_deno_dir')
```

Comment:
This is just any path that doesn't exist, right? Rather than a subdirectory of the current DENO_DIR (which is a bit confusing) can you just do something like `some_path_that_doesnt_exist`?

Reply:
Since the path _does_ get created by the current (fixed) code when the repl exits, I wanted to make sure the path was somewhere that creating it wouldn't be a problem.